adhocteam/Head-Start-TTADP
# Frontend tweaks

**Description of change**
 * Selected link is bold
 * More spacing on the left site nav
 * Secondary links in left site nav are dimmed
 * TTA Provided field is now textarea, helps when user is entering large amount of text
 * Resource selector has more left margin
 * Report message tweaked to say "Activity report for region {region}"
 * Landing page has some more left margin
 * Report submitted message is now marked as "alert" so should be read by screen readers properly

**How to test**

Pull down and test locally or go to [sandbox](https://tta-smarthub-sandbox.app.cloud.gov/)

The alert read properly after submitting a report but if someone with a popular screen reader could double check that would be appreciated.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/338
* https://github.com/HHS/Head-Start-TTADP/issues/383

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
